src/System.Private.CoreLib/shared/System/MemoryExtensions.Fast.cs

```diff
@@ -14,7 +14,7 @@ namespace System
     /// <summary>
     /// Extension methods for Span{T}, Memory{T}, and friends.
     /// </summary>
-    public static partial class MemoryExtensions
+    partial class MemoryExtensions // .Fast
```

Comment:
Why remove `public static`?

Reply:
The main class (`MemoryExtensions.cs`) already declares all the accessors, `public static`. The secondary classes thus don't need to redundantly add them.
I used this pattern on the new class (`.Trim`), so I aligned the existing one (`.Fast`) too.

Reply:
Let me know if you still want me to revert

Reply:
I, personally, would have kept `public static` to be explicit/clear especially as these sources get mirrored and moved/conditionally included (in both places). If we have existing patterns like this in the repo where we drop the access modifier/etc., then we can keep it as is.

Reply:
Just a **quick search**, we should be explicit:
https://github.com/dotnet/coreclr/blob/030a3ea9b8dbeae89c90d34441d4d9a1cf4a7de6/src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.FileAttributes.cs#L7

https://github.com/dotnet/coreclr/blob/030a3ea9b8dbeae89c90d34441d4d9a1cf4a7de6/src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.FileTypes.cs#L7

Reply:
OK, no problem, I will revert